exercise/models.py

```diff
@@ -22,8 +22,15 @@
 )
 
 
-class ExerciseAttachment(PhishtrayBaseModel):
+class ExerciseFile(PhishtrayBaseModel):
+    """
+    ExercieseFiles are not actual files but they act like one.
+    """
     filename = models.CharField(max_length=250, blank=True, null=True)
+    description = models.CharField(max_length=250, blank=True, null=True)
+    # img_url - used for file representation (e.g.: screenshot of the file)
+    # Provide default image?
+    img_url = models.CharField(max_length=500, blank=True, null=True)
```

Comment:
Why not use an ImageField? Unless we don't want to store an actual image here?

Reply:
Yeah, there's no need - it's just a mock to represent a file in the exercise and not an actual file.

Reply:
Yeah that's what I was thinking might be the case but I wasn't sure. :)

Reply:
:) 👍